scripts/fresh_wheel_smoke.py

```
@@ -0,0 +1,380 @@
+                    raise OSError("verified snapshot write made no progress")
+                remaining = remaining[written:]
+        os.fsync(destination_fd)
+        after = os.fstat(source_fd)
+        identity = lambda value: (
+            value.st_dev,
+            value.st_ino,
+            value.st_size,
+            value.st_mtime_ns,
+            value.st_ctime_ns,
+            value.st_nlink,
+        )
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `identity` as a `def`

(E731)